app/views/bags/index.html.erb

```diff
@@ -5,8 +5,10 @@
   <% @bags.each do |bag| %>
     <div>
       <%= link_to bag.name, bag_path(bag) %>
+      <% if @user == bag.user %>
```

Comment:
are we able to use current_user directly here? instead of creating @user in controller and using here

Reply:
Yes sir, should be dry as hell in next commit